10_2_es_bipedal.py

```diff
@@ -0,0 +1,82 @@
+# Evolution Strategies BipedalWalker-v2
+# https://blog.openai.com/evolution-strategies/
+# converges at around iter 100 in 5 minutes
+# for testing model set reload=True
+
+import gym
+import numpy as np
+import cPickle as pickle
+import sys
+
+env = gym.make('BipedalWalker-v2')
+np.random.seed(10)
+
+hl_size = 100
```

Comment:
Can you add comments for each hyper parameters?
Also please follow some standard naming conventions.
`aver_reward` should be renamed to something like `avg_reward`